### PR TITLE
chore(): v9.0.0 release (wip)

### PR DESCRIPTION
**Pull request by kamilmysliwiec**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/nest/blob/master/CONTRIBUTING.md
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Other... Please describe: Major release  

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

PRs:

- https://github.com/nestjs/nest/pull/9604
- https://github.com/nestjs/nest/pull/9587
- https://github.com/nestjs/nest/pull/9586
- https://github.com/nestjs/nest/pull/9534
- https://github.com/nestjs/nest/pull/9507
- https://github.com/nestjs/nest/pull/9491
- https://github.com/nestjs/nest/pull/9383
- https://github.com/nestjs/nest/pull/9311
- https://github.com/nestjs/nest/pull/9230
- https://github.com/nestjs/nest/pull/9023
- https://github.com/nestjs/nest/pull/8798
- https://github.com/nestjs/nest/pull/8459
- https://github.com/nestjs/nest/pull/8058
- https://github.com/nestjs/nest/pull/9596
- https://github.com/nestjs/nest/pull/9591
- https://github.com/nestjs/nest/pull/9590
- https://github.com/nestjs/nest/pull/9697
- https://github.com/nestjs/nest/pull/9684
- https://github.com/nestjs/nest/pull/9681
- https://github.com/nestjs/nest/pull/9674
- https://github.com/nestjs/nest/pull/9648
- https://github.com/nestjs/nest/pull/9718
- https://github.com/nestjs/nest/pull/9705
- https://github.com/nestjs/nest/pull/8802

## Does this PR introduce a breaking change?
- [x] Yes
- [ ] No

## Other information

### CODE REVIEW

**Comment by Zclhlmgqzc** [comment]
ref https://github.com/nestjs/nest/issues/8108#issuecomment-935621712
I will create a PR about middleware for v9

File: packages/platform-fastify/package.json [hunk 1]
```diff
@@ -17,16 +17,26 @@
     "access": "public"
   },
   "dependencies": {
-    "fastify": "3.29.0",
-    "fastify-cors": "6.1.0",
-    "fastify-formbody": "5.3.0",
+    "@fastify/cors": "7.0.0",
```
Comment (Eomm) [inline packages/platform-fastify/package.json:20]: v7 supports fastify v3

```suggestion
    "@fastify/cors": "8.0.0",
```

**Comment by yufengwang** [comment]
Hi, why this pr is labed Draft, and when will nextjs version 9 be final released ? thanks

**Comment by yufengwang** [comment]
Great work, but when  the new npm package will be published ? thank you

**Pull request merged**